extras/shell/src/main/java/org/apache/rya/shell/RyaAdminCommands.java

```diff
@@ -320,11 +343,14 @@ public String installWithMongoParameters(
             final boolean enableFreeTextIndex,
 
             // TODO RYA-215
-//            @CliOption(key = {"enableGeospatialIndex"}, mandatory = false, help = "Use Geospatial Indexing.", unspecifiedDefaultValue = "false", specifiedDefaultValue = "true")
-//            final boolean enableGeospatialIndex,
+            //            @CliOption(key = {"enableGeospatialIndex"}, mandatory = false, help = "Use Geospatial Indexing.", unspecifiedDefaultValue = "false", specifiedDefaultValue = "true")
+            //            final boolean enableGeospatialIndex,
 
             @CliOption(key = {"enableTemporalIndex"}, mandatory = false, help = "Use Temporal Indexing.", unspecifiedDefaultValue = "false", specifiedDefaultValue = "true")
-            final boolean enableTemporalIndex) {
+            final boolean enableTemporalIndex,
```

Comment:
Since you added entity stuff back in, also add it here?

Reply:
I went ahead and just reverted it, was faster to just take care of it later